documentation/src/main/markdown/packagemgrs/python.md

```diff
@@ -16,6 +17,19 @@
 	* Pip Requirements File Parse
 * Poetry detector
 
+## Setuptools detector
+
+Setuptools detector attempts to run on your project if a pyproject.toml file containing a build section with `requires = ["setuptools"]` or equivalent line is located, and a pip installation is found. (Setuptools scans can be run in both build, if a pip installation is available, and buildless mode, if not.)
+
+<note type="note">Setuptools build detector should be run in a virtual environment, or environment with a clean global pip cache, where a pip install has only been performed for the project being scanned.</note>
+
+[company_name] [solution_name] parses the pyproject.toml file determining if the `[build-system]` section has been configured for Setuptools via the `requries= ["setuptools"]` setting. If the setting is located and pip is installed in the environment, either in the default location or specified via the `--detect.pip.path` property, the detector will execute in a virtual environment and analyze the pyproject.toml, setup.cfg, or setup.py files for dependencies. If the detector discovers a configured pyproject.toml file but not a pip executible, it will execute in buildless mode where it will parse dependencies from the pyproject.toml, setup.cfg, or setup.py files but may not be able to specify exact package versions. If no dependencies are located in the pyproject.toml, setup.cfg, or setup.py files, or if the detector fails, the BDIO file output will not be generated in build or buildless mode. [company_name] [solution_name] will also attempt to run additional detectors if their execution requirements are met.
```

Comment:
This is probably okay, going to approve but just calling out that the detector will not run in a virtual environment unless the user has taken steps to manually activate that environment and then run detect in it.

Reply:
I made a tweak. Thank you.